### PR TITLE
Update trytravis.py

### LINKED ISSUE #17: Output Travis URL and stop
Is there any way to stop the wait until the builds are done? I'd like to output the matrix and the url but not wait until all the builds are done. Amazing work btw!

### PR DESCRIPTION
**Pull request by miguelbalparda**
This PR should give trytravis the option to just submit the tests to Travis and don't wait until they are done.
Fixes #17

### CODE REVIEW

File: trytravis.py [hunk 1]
```diff
@@ -367,6 +367,12 @@ def _main(argv):
                 url = None
             _input_github_repo(url)
 
+        # No wait
+        elif arg in ['--no-wait', '-nw']:
+            url = _load_github_repo()
+            commit, committed = _submit_changes_to_github_repo(os.getcwd(), url)
```
Comment (sethmlarson) [inline trytravis.py]: This line is a little too long and is tripping our linter.

**Review by sethmlarson — CHANGES_REQUESTED** [review]
Thanks for submitting this! I have one comment. Also could we also have a CHANGELOG entry under `Added`?

**Comment by sethmlarson** [comment]
Might want to change your `trytravis` target repo from your fork to a different GitHub repository. :)

**Comment by miguelbalparda** [comment]
I was actually trying to run trytravis for my fork to stop all the unnecesary noise here, sorry about that.

**Comment by miguelbalparda** [comment]
How do you want to handle the CHANGELOG entry? Should I add it under the current version? What about the codecov fails?

**Comment by sethmlarson** [comment]
You can add it under a new version labeled "Unreleased" or under the next incremented minor. Don't worry about the codecov fails.

**Comment by miguelbalparda** [comment]
Sorry for the late reply, just added this to the README.md

**Comment by sethmlarson** [comment]
Thanks! This is looking good :)

**Pull request merged**